### ISSUE TITLE
canUpdate?

### ISSUE THREAD

**Issue by hippich** [leeluolee/stateman#26]
there is canEnter and canLeave, but no canUpdate as far as I can see in documentation. It would be useful to have it for cases where you just switch `:id` within same state and you need check if user has access to it.

**Comment by leeluolee** [comment]
Yes , 	that's just what i was thinking.  It will be introduced at next version

**Comment by leeluolee** [comment]
@hippich  I'm going to supprt `canUpdate` now,   The demand for this feature become more strong.

**Comment by hippich** [comment]
@leeluolee, does it mean you already working on it? I could not get started on it yet due vacation time.

**Comment by leeluolee** [comment]
@hippich 

Yes, I will implement it on branch develop . 

Have a Happy Vacation.
